testing/data/EuTaxonomyForNonFinancialsMocks.json

```diff
@@ -1055,7 +1070,7 @@
             },
             "comment": "reboot haptic pixel",
             "value": 8509779631,
-            "unit": "ALL"
```

Comment:
There are still some dataSource entries in this file that use the old naming convention (e.g. "report":)

Reply:
Changed